src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -0,0 +1,49 @@
+package bit.javaoop;
+
+public class FixedBudgetManager extends Manager {
+
+    private double fixedBudget;
+
+    public FixedBudgetManager(String name, double salary) {
+        super(name, salary);
+    }
+
+    public boolean isSatisfied() {
+        if (this.getSalary() > 20_000 && (this.currentBudget() < 15_000)) {
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public boolean hireNewEmployee(Employee employee) {
+        if ((this.currentStuffSalary() + employee.getSalary()) <= this.getFixedBudget()) {
+            this.getMyStaffList().add(employee);
+            return true;
+        } else {
+            System.out.println("You don't have enought money to hire new employee.");
+            return false;
+        }
+    }
+
+    public double currentStuffSalary() {
+        double sum = 0;
+        for (Employee employee : this.getMyStaffList()) {
+            sum += employee.getSalary();
+        }
+        return sum;
+    }
+
+    public double currentBudget() {
```

Comment:
Takie metody pomocniczne to:
- Powinny być prywatne.
- Powinny być jak najbliżej w kodzie metod, które ich używają i przede wszystkim pod nimi.